src/CarbonAware.CLI/src/CarbonAware.CLI.csproj

```diff
@@ -52,11 +52,19 @@
   </ItemGroup>
 
  <Target Name="CopyDataFiles" AfterTargets="Build">
-		<ItemGroup>
-			<DataFiles Include="$(ProjectDir)..\data\data-files\*.*" />
-		</ItemGroup>
+    <ItemGroup>
+      <DataFiles Include="$(ProjectDir)..\..\data\data-files\**\*.*" />
```

Comment:
This may be my .NET ignorance, but couldn't we include the data source projects `<ProjectReference Include="..\..\CarbonAware.DataSources\CarbonAware.DataSources.Json\src\CarbonAware.DataSources.Json.csproj" />` and let the project manage it's own file expectations?

Reply:
Not sure I get your question. Maybe we can sync offline?